src/public/services/PublicFormAuthService.ts

```diff
@@ -13,12 +13,13 @@ export const PUBLIC_FORMS_ENDPOINT = '/api/v3/forms'
 export const createRedirectURL = (
   formId: string,
   isPersistentLogin = false,
+  encodedQuery = '',
```

Comment:
should we allow this to be `undefined` instead of setting it to an empty string?

```
encodedQuery?: string
```